vue-frontend/public/robots.txt

```diff
@@ -0,0 +1,4 @@
+User-agent: *
```

Comment:
can you add reason in description of adding robots.txt file?
And what it will do?